index.html

```diff
@@ -229,26 +229,15 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital
-signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This specification defines how to express proofs of integrity for verifiable
+credentials using a number of concrete serializations for each of the defined
+syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group.
               </p>
+              <p class="draft-status"><b>Draft state:</b> No Draft </p>
 
-              <p class="draft-status"><b>Draft state:</b>
+              <p class="input-documents"><b>Input documents:</b></p>
+                <p>(the following are a non-exhaustive selection of expected input documents)</p>
                 <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
```

Comment:
A nit to be cleaned up later, but I think ul / li would be more readable than a comma separated list.